src/PrestaShopBundle/Entity/Repository/LogRepository.php

```diff
@@ -187,6 +188,8 @@ public function deleteAll()
      */
     public function getSearchQueryBuilder(SearchCriteriaInterface $searchCriteria)
     {
+        @trigger_error(sprintf('The "%s()" method is deprecated since 1.7.8.0 See LogQueryBuilder', __METHOD__), E_USER_DEPRECATED);
```

Comment:
@PululuK docblock deprecated is not needed?

Reply:
Done